agent/nswrapper/ns.go

```diff
@@ -0,0 +1,25 @@
+package nswrapper
```

Comment:
this can be under utils/nswrapper to be consistent with few other wrappers

Reply:
Will move.